colin/core/ruleset/ruleset.py

```diff
@@ -206,3 +206,13 @@ def get_ruleset_directory():
     msg = "Ruleset directory cannot be found."
     logger.warning(msg)
     raise ColinRulesetException(msg)
+
+
+def get_rulesets():
+    """"
+    Get available rulesets.
+    """
+    rulesets_dir = get_ruleset_directory()
+    ruleset_files = [os.path.basename(f)[:-5] for f in os.listdir(rulesets_dir) if
+                     os.path.isfile(os.path.join(rulesets_dir, f)) and f.lower().endswith(JSON)]
```

Comment:
Hehe, I've just implemented it myself, but you've been quicker.
Anyway I'd prefer `len(JSON)` if that's what the `5` means.
Also the `basename()` is I think not necessary, but does no harm, so OK to me.

Reply:
Sorry..;-) (I will use that, so it will be more readable.)

Reply:
LGTM.
BTW, do you know [glob](https://docs.python.org/3/library/glob.html#glob.glob) ?
```
ruleset_files = [os.path.basename(f)[:-len(JSON)].lower() for f in glob.glob(rulesets_dir + '/*' + JSON)]
```